mongospecs/mixins/crud.py

```
@@ -0,0 +1,323 @@
+        signal("deleted").send(self.__class__, specs=[self])
+
+    @classmethod
+    def find(cls, filter: FilterType = None, **kwargs: t.Any) -> list[SpecDocumentType]:
+        """Return a list of documents matching the filter"""
+        # Flatten the projection
+        kwargs["projection"], references, subs = cls._flatten_projection(
+            kwargs.get("projection", cls._default_projection)
+        )
+
+        # Find the document
+        if isinstance(filter, (Condition, Group)):
```

Ruff (0.8.2)

Undefined name `Condition`

(F821)